docs/development/contributor_guidelines.rst

```diff
@@ -33,18 +33,27 @@ Code Style
 ----------
 
 We use code linters to keep a consistent code quality and style. These linters
-also run in CI and will produce build failures. To avoid this, we suggest running
-the linters locally as you work.
+also run in CI and will produce build failures. To avoid this, we have included
+a git pre-commit hook. You can install it with the following command run at the
+root of the repository:
+
+.. code:: sh
+
+    ln -sf git/pre-commit .git/hooks/pre-commit
```

Comment:
Symlink won't work with this syntax, since the target must be relative. Ideally this would work:

```
ln -sf ../../git/pre-commit .git/hooks/pre-commit
```

but then the Makefile target can't be found, likely due to the filepath madness in the while loop. So just change it back to a `cp` and I'll look into the specifics on a rainy day. Would much rather have a `cp`-able version available for everyone to use pronto.

Reply:
I just pushed a new version. It work with a symlink and plays nicely with the paths.